ReactWindows/ReactNative.Shared/UIManager/BorderedCanvasManager.cs

```diff
@@ -12,16 +13,37 @@ namespace ReactNative.UIManager
     /// View parent manager for bordered canvases.
     /// </summary>
     public abstract class BorderedCanvasManager<TCanvas> : BorderedViewParentManager<TCanvas>
-        where TCanvas : Canvas
+        where TCanvas : BorderedCanvas
```

Comment:
Fixed. I had to move SetDimensions to ReactViewManager as it changes the border size.